typescript/infra/config/environments/mainnet3/warp/configGetters/getPulsechainUSDCWarpConfig.ts

```diff
@@ -32,13 +28,10 @@ const ownersByChain: DeploymentChains<Address> = {
   pulsechain: '0x703cf58975B14142eD0Ba272555789610c85520c',
 };
 
-const rebalancingConfigByChain = getUSDCRebalancingBridgesConfigFor(
-  Object.keys(ownersByChain),
-);
-
-const getRebalanceableCollateralTokenConfigForChain = (
-  currentChain: CollateralChain,
+export const getRebalanceableCollateralTokenConfigForChain = (
```

Comment:
maybe it would be a good idea to move the `getRebalanceableCollateralTokenConfigForChain` and `getSyntheticTokenConfigForChain` to a separate file so they can be reused for other confg getters without having to import from the pulsechain file